lib/tasks/delete_routes.rake

```diff
@@ -0,0 +1,28 @@
+#
+#collections responding 301
+#collections = ["/government/collections/ad-hoc-statistical-analysis-2015-quarter-1", "/government/collections/cde-marketplace-5-february-2015-exhibitor-case-studies", "/government/collections/chapter-32-port-cases-involving-prosecution-immigration-directorate-instructions", "/government/collections/command-papers", "/government/collections/departmental-exceptions-to-spending-controls-2014", "/government/collections/electronic-business-commissioners-directions", "/government/collections/flagging-up-newsletters", "/government/collections/green-deal-quick-guides", "/government/collections/greenhouse-gas-conversion-factors-for-company-reporting", "/government/collections/guidance-on-british-citizenship", "/government/collections/house-of-commons-papers", "/government/collections/ministerial-gifts-hospitality-travel-and-meetings-2012", "/government/collections/ministerial-gifts-hospitality-travel-and-meetings-2013", "/government/collections/national-curriculum-assessments-2013", "/government/collections/official-documents", "/government/collections/oisc-news", "/government/collections/self-assessment-helpsheets-additional-information", "/government/collections/social-care-online-questionnaires-2015", "/government/collections/think-act-report-sign-ups-and-case-studies"]
+
+desc "Delete Routes"
+task :delete_routes, [:paths_type] => :environment do |t, args|
```

Comment:
Could you change the indentation to at least be consistent (the top half uses a single tab, the last two lines use two spaces), and preferably use 2 spaces throughout as per the ruby styleguide [1].

1: https://github.com/alphagov/styleguides/blob/master/ruby.md